minicriu-client.c

```diff
@@ -46,16 +46,26 @@
 #include "minicriu-client.h"
 
 #define MC_THREAD_SIG SIGSYS
+#define MAX_MAPS 512
 
 static volatile uint32_t mc_futex_checkpoint;
 static volatile uint32_t mc_futex_restore;
+static volatile uint32_t restored_threads;
```

Comment:
There is a convention for global (static) functions/variables to start with `mc_`